src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -129,6 +148,23 @@ def GetClientStats(self, request: messages_pb2.LoadBalancerStatsRequest,
         logger.info("Returning stats response: {}".format(response))
         return response
 
+    def GetClientAccumulatedStats(
+            self, request: messages_pb2.LoadBalancerAccumulatedStatsRequest,
+            context: grpc.ServicerContext):
+        logger.info("Received cumulative stats request.")
+        response = messages_pb2.LoadBalancerAccumulatedStatsResponse()
+        with _global_lock:
+            for method in _SUPPORTED_METHODS:
+                caps_method = _METHOD_CAMEL_TO_CAPS_SNAKE[method]
+                response.num_rpcs_started_by_method[
+                    caps_method] = _global_rpcs_started[method]
+                response.num_rpcs_succeeded_by_method[
+                    caps_method] = _global_rpcs_succeeded[method]
+                response.num_rpcs_failed_by_method[
+                    caps_method] = _global_rpcs_failed[method]
+        logger.info("Returning cumulative stats response.")
```

Comment:
optional: Will it be nice to briefly log the 3 stats above?

Reply:
This is already being logged by the test runner. Based on how the logs are presented, that's probably the best place for that information to show up.